src/flair/predictProductivity.py

```diff
@@ -265,22 +265,43 @@ def get_exons(bedline):
 
 
 def predict(bed, starts, isoDict, nmdexcep):
+    fusiondict = {}
     for line in open(bed):
         line = line.rstrip().split('\t')
         read   = line[3]
```

Comment:
read_id would be a better name, at first I thought this was the actual read bed record